FrontDesk/src/FrontDesk.Api/Endpoints/Configuration/Read.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Ardalis.ApiEndpoints;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace FrontDesk.Api.ConfigurationEndpoints
+{
+    public class Read : BaseAsyncEndpoint<string, string>
+    {
+        public Read()
+        {
+        }
+
+        [HttpGet("api/configurations/{configurationName}")]
```

Comment:
We don't have that much configuration - wouldn't it make sense to have the endpoint return all config values needed by the client app?

Reply:
Done